bitwarden_license/src/Scim/Controllers/v2/UsersController.cs

```diff
@@ -38,22 +42,26 @@ public class UsersController : Controller
         _organizationService = organizationService;
         _scimContext = scimContext;
         _scimSettings = scimSettings?.Value;
+        _getUserCommand = getUserCommand;
         _logger = logger;
     }
 
     [HttpGet("{id}")]
     public async Task<IActionResult> Get(Guid organizationId, Guid id)
     {
-        var orgUser = await _organizationUserRepository.GetDetailsByIdAsync(id);
-        if (orgUser == null || orgUser.OrganizationId != organizationId)
+        try
         {
-            return new NotFoundObjectResult(new ScimErrorResponseModel
+            var scimUserResponseModel = await _getUserCommand.GetUserAsync(organizationId, id);
+            return Ok(scimUserResponseModel);
+        }
+        catch (NotFoundException ex)
+        {
+            return NotFound(new ScimErrorResponseModel
             {
-                Status = 404,
-                Detail = "User not found."
+                Status = StatusCodes.Status404NotFound,
```

Comment:
`NotFound` returns `NotFoundObjectResult` which is described as having a status code of 404 by default. Do we need to assign a `Status` here?

Reply:
The status set here is for the returned body content.

Reply:
Ah, good point